avm/res/container-service/managed-cluster/main.bicep

```diff
@@ -297,6 +297,12 @@ param enableWorkloadIdentity bool = false
 @description('Optional. Whether to enable Azure Defender.')
 param enableAzureDefender bool = false
 
+@description('Optional. Whether to enable Image Cleaner for Kubernetes.')
+param enableImageCleaner bool = false
+
+@description('Optional. The interval in hours Image Cleaner will run. Defaults 120, 5 days.')
+param imageCleanerIntervalHours int = 120
```

Comment:
The default interval hours when using Azure CLI is one week (168 hours). The minimum is 24 hours. 

I think we should set this to 24 hours by default. 

https://learn.microsoft.com/en-us/azure/aks/image-cleaner#configuration-options

CC: @JPEasier

Reply:
Happy to change it if needed. I just used the value I use internally. I found it a good fit.